cogs/resourcecog.py

```diff
@@ -0,0 +1,169 @@
+from discord.ext import commands
+import discord
+import json
+from discord.utils import get
+
+
+def jsonsave(cfg):
+    with open("data/resources.json", "w") as f:
+        json.dump(cfg, f)
+
+
+def jsonopen():
+    with open("data/resources.json", "r") as f:
+        return json.load(f)
+
+
+def create_embed(msgauth, name, link, desc, tags):
+    embed = discord.Embed(
+        title="Resource Suggestion",
+        description="Suggestion made by " + msgauth,
+        color=0x66FF99,
+    )
+
+    embed.add_field(name="Resource Name", value=name, inline=False)
+
+    embed.add_field(name="Link", value=link, inline=False)
+
+    embed.add_field(name="Description", value=desc, inline=False)
+
+    embed.add_field(name="Tags", value=tags, inline=False)
+    return embed
+
+
+def edit_embed(rinfo):
+    return create_embed(
+        rinfo["msgauth"], rinfo["name"], rinfo["link"], rinfo["desc"], rinfo["tags"]
+    )
+
+
+async def send_resource(rsc_channel, msg):
+    await msg.delete()
+    cfg = jsonopen()
+    rinfo = cfg[str(msg.id)]
+    rscembed = discord.Embed(
+        title=rinfo["name"], description=rinfo["desc"], color=0x66FF99
+    )
+    rscembed.add_field(name="Link", value=rinfo["link"], inline=True)
+    rscembed.add_field(name="Tags", value=rinfo["tags"], inline=True)
+    rscembed.set_footer(text="Submitted by " + rinfo["msgauth"])
+    await rsc_channel.send(embed=rscembed)
+    del cfg[str(msg.id)]
+    jsonsave(cfg)
+
+
+class ResourceCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+        with open("config.json", "r") as f:
+            cfg = json.load(f)
```

Comment:
It is a little confusing that you use `cfg` as a variable name both when actually referring to the configuration file object and when referring to other JSON objects. Here it should stay, but in other places where you use `cfg` as a stand in for something like "resource dictionary" please choose a more clear name for the object you're dumping or loading from a json file, such as `resource_dict` or something.